homeassistant/components/verisure/alarm_control_panel.py

```diff
@@ -81,8 +86,10 @@ async def _async_set_arm_state(
                 .get("armStateChangePollResult", {})
                 .get("result")
             )
-
-        await self.coordinator.async_refresh()
```

Comment:
I think it's weird, in your PR description you mention that you want your update to go sooner than the requested update, but what @zweckj mentions is that requesting the coordinator for an update already does this, as we fetch new state. I did not really see an argument against why this wouldn't work

Reply:
Okay but it isn't clear to me why we shouldn't pull directly after doing something?

Reply:
Because we already have the data, the essense of @zweckj's comment

Reply:
Because we already have the data, the essense of @zweckj's comment